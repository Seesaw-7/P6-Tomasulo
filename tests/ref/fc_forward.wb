PC=00000000, ---
PC=00000004, REG[ 1]=00000004
PC=00000008, REG[ 1]=00000090
PC=0000000c, REG[ 2]=0001000c
PC=00000010, REG[ 2]=00010000
PC=00000014, REG[ 8]=00010000
PC=00000018, REG[ 3]=00001018
PC=0000001c, REG[ 3]=00001120
PC=00000020, REG[ 4]=00000000
PC=00000024, REG[ 5]=00000000
PC=00000028, REG[ 6]=00000000
PC=0000002c, REG[ 7]=00000000
PC=00000030, REG[ 9]=00000000
PC=00000034, REG[10]=00000000
PC=00000038, REG[11]=00000000
PC=0000003c, REG[12]=00000000
PC=00000040, REG[13]=00000000
PC=00000044, REG[14]=00000000
PC=00000048, REG[15]=00000000
PC=0000004c, REG[16]=00000000
PC=00000050, REG[17]=00000000
PC=00000054, REG[18]=00000000
PC=00000058, REG[19]=00000000
PC=0000005c, REG[20]=00000000
PC=00000060, REG[21]=00000000
PC=00000064, REG[22]=00000000
PC=00000068, REG[23]=00000000
PC=0000006c, REG[24]=00000000
PC=00000070, REG[25]=00000000
PC=00000074, REG[26]=00000000
PC=00000078, REG[27]=00000000
PC=0000007c, REG[28]=00000000
PC=00000080, REG[29]=00000000
PC=00000084, REG[30]=00000000
PC=00000088, REG[31]=00000000
PC=0000008c, ---
PC=00000188, REG[ 2]=0000ff90
PC=0000018c, ---
PC=00000190, ---
PC=00000194, ---
PC=00000198, ---
PC=0000019c, ---
PC=000001a0, ---
PC=000001a4, ---
PC=000001a8, ---
PC=000001ac, ---
PC=000001b0, ---
PC=000001b4, ---
PC=000001b8, ---
PC=000001bc, ---
PC=000001c0, REG[ 8]=00010000
PC=000001c4, REG[ 5]=0000ff90
PC=000001c8, REG[ 9]=0000ff90
PC=000001cc, REG[ 5]=00001000
PC=000001d0, REG[ 5]=00000004
PC=000001d4, REG[ 1]=00000003
PC=000001d8, ---
PC=000001dc, REG[ 1]=00000004
PC=000001e0, REG[26]=00000004
PC=000001e4, REG[27]=00000000
PC=000001e8, REG[ 1]=00000000
PC=000001ec, REG[29]=00000000
PC=000001f0, REG[29]=00000000
PC=000001f4, REG[28]=00000080
PC=000001f8, REG[28]=00000004
PC=000001fc, REG[24]=00000004
PC=00000200, REG[25]=00000000
PC=00000204, REG[28]=00000000
PC=00000208, REG[ 7]=00000000
PC=0000020c, REG[ 7]=00000000
PC=00000210, REG[ 6]=00000080
PC=00000214, REG[ 6]=00000004
PC=00000218, REG[ 6]=00000010
PC=0000021c, REG[ 6]=0000001f
PC=00000220, REG[ 6]=00000001
PC=00000224, REG[ 6]=00000010
PC=00000228, REG[ 2]=0000ff80
PC=0000022c, REG[ 6]=0000ff80
PC=00000230, REG[ 6]=0000ff83
PC=00000234, REG[ 6]=00003fe0
PC=00000238, REG[ 6]=0000ff80
PC=0000023c, ---
PC=00000240, REG[ 6]=00001000
PC=00000244, REG[ 6]=00000004
PC=00000248, REG[28]=00000003
PC=0000024c, ---
PC=00000250, REG[28]=00000004
PC=00000254, REG[22]=00000004
PC=00000258, REG[23]=00000000
PC=0000025c, REG[28]=00000000
PC=00000260, REG[17]=00000000
PC=00000264, REG[17]=00000000
PC=00000268, REG[16]=00000080
PC=0000026c, REG[16]=00000004
PC=00000270, REG[20]=00000004
PC=00000274, REG[21]=00000000
PC=00000278, REG[16]=00000000
PC=0000027c, REG[11]=00000000
PC=00000280, REG[11]=00000000
PC=00000284, REG[10]=00000080
PC=00000288, REG[11]=00000004
PC=0000028c, REG[11]=00000010
PC=00000290, REG[11]=0000001f
PC=00000294, REG[11]=00000001
PC=00000298, REG[11]=00000010
PC=0000029c, REG[ 2]=0000ff70
PC=000002a0, REG[11]=0000ff70
PC=000002a4, REG[11]=0000ff73
PC=000002a8, REG[11]=00003fdc
PC=000002ac, REG[11]=0000ff70
PC=000002b0, ---
PC=000002b4, REG[11]=00001000
PC=000002b8, REG[11]=00000004
PC=000002bc, REG[10]=00000003
PC=000002c0, ---
PC=000002c4, REG[10]=00000004
PC=000002c8, REG[18]=00000004
PC=000002cc, REG[19]=00000000
PC=000002d0, REG[10]=00000000
PC=000002d4, REG[13]=00000000
PC=000002d8, REG[13]=00000000
PC=000002dc, REG[12]=00000080
PC=000002e0, REG[13]=00000004
PC=000002e4, REG[30]=00000004
PC=000002e8, REG[31]=00000000
PC=000002ec, REG[13]=00000000
PC=000002f0, REG[15]=00000000
PC=000002f4, REG[15]=00000000
PC=000002f8, REG[14]=00000080
PC=000002fc, REG[15]=00000004
PC=00000300, REG[15]=00000010
PC=00000304, REG[15]=0000001f
PC=00000308, REG[15]=00000001
PC=0000030c, REG[15]=00000010
PC=00000310, REG[ 2]=0000ff60
PC=00000314, REG[15]=0000ff60
PC=00000318, REG[15]=0000ff63
PC=0000031c, REG[15]=00003fd8
PC=00000320, REG[15]=0000ff60
PC=00000324, ---
PC=00000328, ---
PC=0000032c, ---
PC=000003a8, REG[15]=00001000
PC=000003ac, REG[15]=00000004
PC=000003b0, REG[14]=00000000
PC=000003b4, ---
PC=00000330, REG[ 1]=00000330
PC=00000334, REG[ 1]=00000338
