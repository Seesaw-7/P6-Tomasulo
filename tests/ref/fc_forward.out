@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 0000009700000013 : 648540061715
@@@ mem[    8] = 0001011708c08093 : 282673419419795
@@@ mem[   16] = 00010413ff410113 : 285958910050579
@@@ mem[   24] = 1081819300001197 : 1189374244962701719
@@@ mem[   32] = 0000029300000213 : 2830383448595
@@@ mem[   40] = 0000039300000313 : 3929895076627
@@@ mem[   48] = 0000051300000493 : 5579162518675
@@@ mem[   56] = 0000061300000593 : 6678674146707
@@@ mem[   64] = 0000071300000693 : 7778185774739
@@@ mem[   72] = 0000081300000793 : 8877697402771
@@@ mem[   80] = 0000091300000893 : 9977209030803
@@@ mem[   88] = 00000a1300000993 : 11076720658835
@@@ mem[   96] = 00000b1300000a93 : 12176232286867
@@@ mem[  104] = 00000c1300000b93 : 13275743914899
@@@ mem[  112] = 00000d1300000c93 : 14375255542931
@@@ mem[  120] = 00000e1300000d93 : 15474767170963
@@@ mem[  128] = 00000f1300000e93 : 16574278798995
@@@ mem[  136] = 0fc0006f00000f93 : 1134907582838738835
@@@ mem[  144] = f701011300010117 : 17798508383460983063
@@@ mem[  152] = 00000013fea12c23 : 85876354083
@@@ mem[  160] = 0000001310500073 : 81878057075
@@@ mem[  168] = 0000001300000013 : 81604378643
@@@ mem[  176] = 02812e23fc010113 : 180476192158187795
@@@ mem[  184] = fca42e2304010413 : 18204726321664295955
@@@ mem[  192] = fcc42a23fcb42c23 : 18213729127045016611
@@@ mem[  200] = fce42623fcd42823 : 18222731928255342627
@@@ mem[  208] = fe042223fe042623 : 18303792323521422883
@@@ mem[  216] = fe042423fe042023 : 18303794522544676899
@@@ mem[  224] = 0700006ffe042623 : 504403639268550179
@@@ mem[  232] = 00279793fec42783 : 11144185981970307
@@@ mem[  240] = 00f707b3fdc42703 : 69532788885563139
@@@ mem[  248] = fec427830007a683 : 18357841424663684739
@@@ mem[  256] = fd84270300279793 : 18267768882362554259
@@@ mem[  264] = 0007a70300f707b3 : 2153956179904435
@@@ mem[  272] = 00279793fec42783 : 11144185981970307
@@@ mem[  280] = 00f607b3fd442603 : 69251313900463619
@@@ mem[  288] = 00e7a02302e68733 : 65196791853123379
@@@ mem[  296] = 00279793fec42783 : 11144185981970307
@@@ mem[  304] = 00f707b3fd442703 : 69532788877174531
@@@ mem[  312] = fe8427030007a783 : 18339826476398389123
@@@ mem[  320] = fef4242300f707b3 : 18371348512709740467
@@@ mem[  328] = 00178793fec42783 : 6622994168555395
@@@ mem[  336] = 000017b7fef42623 : 26079023867427
@@@ mem[  344] = fec427039407a783 : 18357840877390899075
@@@ mem[  352] = fe842703f8f744e3 : 18339826480574842083
@@@ mem[  360] = 00f70733fd042783 : 69532239117166467
@@@ mem[  368] = 00e7a023fcc42783 : 65196796045174659
@@@ mem[  376] = 03c1240300000013 : 270537047922442259
@@@ mem[  384] = 0000806704010113 : 141179937161491
@@@ mem[  392] = 06112623f9010113 : 437172574774952211
@@@ mem[  400] = 0691222306812423 : 473196969678939171
@@@ mem[  408] = 05312e2307212023 : 374130972026806307
@@@ mem[  416] = 05512a2305412c23 : 383133773203581987
@@@ mem[  424] = 0571262305612823 : 392136574413908003
@@@ mem[  432] = 0591222305812423 : 401139375624234019
@@@ mem[  440] = 03b12e2305a12023 : 266044580944748579
@@@ mem[  448] = 0001029307010413 : 284305477665811
@@@ mem[  456] = 000012b700028493 : 20577188480147
@@@ mem[  464] = fff280939402a283 : 18442944795367350915
@@@ mem[  472] = 00028093fa142c23 : 704322997595171
@@@ mem[  480] = 00000d9300008d13 : 14925011389715
@@@ mem[  488] = 005d9e9301bd5093 : 26351527060656275
@@@ mem[  496] = 005d1e1301d0eeb3 : 26210239817772723
@@@ mem[  504] = 000e0c1300028e13 : 3953925418028563
@@@ mem[  512] = 01bc5e1300000c93 : 125078325356924051
@@@ mem[  520] = 007e63b3005c9393 : 35575467521905555
@@@ mem[  528] = 00028313005c1313 : 707067587072787
@@@ mem[  536] = 00f3031300231313 : 68401799482250003
@@@ mem[  544] = 0043131300435313 : 18879795769332499
@@@ mem[  552] = 0001031340610133 : 284856196071731
@@@ mem[  560] = 0023531300330313 : 9942965257700115
@@@ mem[  568] = fa642a2300231313 : 18042592336967570195
@@@ mem[  576] = 9403230300001337 : 10665406838335345463
@@@ mem[  584] = fbc42823fff30e13 : 18141669334038285843
@@@ mem[  592] = 000e0b1300030e13 : 3952825906433555
@@@ mem[  600] = 01bb5e1300000b93 : 124796850380213139
@@@ mem[  608] = 011e68b3005b9893 : 80616961353685139
@@@ mem[  616] = 00030813005b1813 : 853302633502739
@@@ mem[  624] = 00000a9300080a13 : 11626476997139
@@@ mem[  632] = 005a959301ba5813 : 25497206525679635
@@@ mem[  640] = 005a151300b865b3 : 25355919264605619
@@@ mem[  648] = 0025959300030593 : 10579032731223443
@@@ mem[  656] = 0045d59300f58593 : 19656600746034579
@@@ mem[  664] = 40b1013300459593 : 4661508407864694163
@@@ mem[  672] = 0035859300010593 : 15065040172418451
@@@ mem[  680] = 002595930025d593 : 10579032733504915
@@@ mem[  688] = 000015b7fab42623 : 23879929308707
@@@ mem[  696] = fff585139405a583 : 18443794168100005251
@@@ mem[  704] = 00058513faa42423 : 1553695739487267
@@@ mem[  712] = 0000099300050913 : 10526965172499
@@@ mem[  720] = 0059969301b95513 : 25216831060530451
@@@ mem[  728] = 0059161300d566b3 : 25075543801423539
@@@ mem[  736] = 00068f1300058693 : 1846161627776659
@@@ mem[  744] = 01bf569300000f93 : 125914503949848467
@@@ mem[  752] = 00f6e7b3005f9793 : 69497600262248339
@@@ mem[  760] = 00058793005f1713 : 1556440319727379
@@@ mem[  768] = 00f7879300279793 : 69673384680069011
@@@ mem[  776] = 004797930047d793 : 20151380967151507
@@@ mem[  784] = 0001079340f10133 : 289804007833907
@@@ mem[  792] = 0027d79300378793 : 11214550455519123
@@@ mem[  800] = faf4222300279793 : 18083115937521178515
@@@ mem[  808] = 07c0006ffa042e23 : 558446834729889315
@@@ mem[  816] = 112080e700000097 : 1234128027525316759
@@@ mem[  824] = 000107b700050713 : 289957537449747
@@@ mem[  832] = 00f77733fff78793 : 69655384468981651
@@@ mem[  840] = fbc42783fb442683 : 18141668646764947075
@@@ mem[  848] = 00f687b300279793 : 69392047142311827
@@@ mem[  856] = 0000009700e7a023 : 648555241507
@@@ mem[  864] = 000507130e6080e7 : 1415153310531815
@@@ mem[  872] = fff78793000107b7 : 18444359864349165495
@@@ mem[  880] = fac4268300f77733 : 18069609949016061747
@@@ mem[  888] = 00279793fbc42783 : 11144185931638659
@@@ mem[  896] = 00e7a02300f687b3 : 65196791820617651
@@@ mem[  904] = fbc42783fa442703 : 18141668646748169987
@@@ mem[  912] = 00f707b300279793 : 69532784630667155
@@@ mem[  920] = fbc427830007a023 : 18141668642549899299
@@@ mem[  928] = faf42e2300178793 : 18083129131659659155
@@@ mem[  936] = 9407a783000017b7 : 10666678423532869559
@@@ mem[  944] = f6f74ee3fbc42703 : 17795779193666283267
@@@ mem[  952] = 08a080e700000097 : 621638478202929303
@@@ mem[  960] = f9c40793faa42023 : 17997518343025991715
@@@ mem[  968] = fa04268300078713 : 18015566753471891219
@@@ mem[  976] = fac42583fa442603 : 18069608853686986243
@@@ mem[  984] = 00000097fb442503 : 652755608835
@@@ mem[  992] = 00000793cd4080e7 : 8331385143527
@@@ mem[ 1000] = 0007851300048113 : 2116641488142611
@@@ mem[ 1008] = 06c12083f9040113 : 486705985923318035
@@@ mem[ 1016] = 0641248306812403 : 450681582882202627
@@@ mem[ 1024] = 05c1298306012903 : 414658283412990211
@@@ mem[ 1032] = 05412a8305812a03 : 378630585897265667
@@@ mem[ 1040] = 04c12b8305012b03 : 342602888381541123
@@@ mem[ 1048] = 04412c8304812c03 : 306575190865816579
@@@ mem[ 1056] = 03c12d8304012d03 : 270547493350092035
@@@ mem[ 1064] = 0000806707010113 : 141179987493139
@@@ mem[ 1072] = 9447a783000017b7 : 10684692822042351543
@@@ mem[ 1080] = 0a07a6230aa7a423 : 722728934649078819
@@@ mem[ 1088] = a803000017b78082 : 12106520223699927170
@@@ mem[ 1096] = 85934c9585b79447 : 9625121033698120775
@@@ mem[ 1104] = 27030a882683f2d5 : 2811102172287333077
@@@ mem[ 1112] = 861302b687b30ac8 : 9661068608623807176
@@@ mem[ 1120] = 242300f637b30017 : 2603926067046711319
@@@ mem[ 1128] = 06135851f6370ac8 : 437790697831467720
@@@ mem[ 1136] = 073302c6863342d6 : 518761433756025558
@@@ mem[ 1144] = 973202b6b6b302b7 : 10894773432335139511
@@@ mem[ 1152] = 0017951397ba9736 : 6637835846850358
@@@ mem[ 1160] = 808281050af82623 : 9260105642486015523
@@@
@@@ mem[ 1248] = 0000000000000500 : 1280
@@@
@@@ mem[ 1280] = 000007ec00000000 : 8710193676288
@@@ mem[ 1288] = 000008bc00000854 : 9603546875988
@@@
@@@ mem[ 1448] = 0000000000000001 : 1
@@@ mem[ 1456] = e66d1234abcd330e : 16603947418614838030
@@@ mem[ 1464] = 0000000b0005deec : 47245025004
@@@
@@@ mem[ 2368] = 0000050000000004 : 5497558138884
@@@
@@@ mem[65440] = 0000ff6000000000 : 280787781943296
@@@ mem[65448] = 0000ff7000000003 : 280856501420035
@@@ mem[65456] = 0000ff8000000003 : 280925220896771
@@@ mem[65464] = 0000000000000003 : 3
@@@
@@@ mem[65528] = 0000009000010000 : 618475356160
@@@
@@@ System halted on illegal instruction
@@@
